core/embed/extmod/modtrezorconfig/modtrezorconfig.c

```diff
@@ -67,30 +67,41 @@ STATIC mp_obj_t mod_trezorconfig_init(size_t n_args, const mp_obj_t *args) {
 STATIC MP_DEFINE_CONST_FUN_OBJ_VAR_BETWEEN(mod_trezorconfig_init_obj, 0, 1,
                                            mod_trezorconfig_init);
 
-/// def unlock(pin: int) -> bool:
+/// def unlock(pin: int, ext_salt: Optional[bytes] = None) -> bool:
 ///     """
-///     Attempts to unlock the storage with given PIN.  Returns True on
-///     success, False on failure.
+///     Attempts to unlock the storage with the given PIN and external salt.
+///     Returns True on success, False on failure.
 ///     """
-STATIC mp_obj_t mod_trezorconfig_unlock(mp_obj_t pin) {
-  uint32_t pin_i = trezor_obj_get_uint(pin);
-  if (sectrue != storage_unlock(pin_i)) {
+STATIC mp_obj_t mod_trezorconfig_unlock(size_t n_args, const mp_obj_t *args) {
```

Comment:
I have a slight preference to make `ext_salt` parameter mandatory (but still with the possibility to provide None as the parameter). This will simplify the code because the number of parameters of this function will be fixed.

Reply:
ACK